lib/view/dailyEventView.dart

```diff
@@ -161,8 +159,8 @@ class _SingleDayState extends State<SingleDay> {
             painter: MyPainter(context, eventSpace: space, event: event),
             child: GestureDetector(
                 behavior: HitTestBehavior.translucent,
-                onTap: () {
-                  showEventDetailPopup(context, event, widget.date);
+                onTap: () async {
```

Comment:
this will be slower and dart doesnt like this but is a consequence of not storing tags in the event/undertaking object and not storing all grabbed events + another db call locally